rotkehlchen/chain/arbitrum_one/modules/gearbox/decoder.py

```diff
@@ -0,0 +1,24 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.decoding.gearbox.decoder import GearboxCommonDecoder
+from rotkehlchen.types import ChainID
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.user_messages import MessagesAggregator
+
+
+class GearboxDecoder(GearboxCommonDecoder):
+    def __init__(
+            self,
+            ethereum_inquirer: 'EthereumInquirer',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: 'MessagesAggregator',
+    ) -> None:
+        super().__init__(
+            evm_inquirer=ethereum_inquirer,
+            base_tools=base_tools,
+            msg_aggregator=msg_aggregator,
+            chain_id=ChainID.ARBITRUM_ONE,
```

Comment:
I just noticed now but you don't need to provide the chainId to the common decoder. You can access it from evm_inquirer.chain_id so you can remove it